### PR TITLE
Add new projects

### PR DESCRIPTION
**Pull request by tasomaniac**
I used the following url to get the new public repos that we have:
https://api.github.com/users/novoda/repos?perpage=100&sort=created

Then I added them to the list of repos. 
When I found appropriate I added some of them to the groups. Can be seen in the diff. 

Comments are welcome.

### CODE REVIEW

File: _coffee/application.coffee [hunk 1]
```diff
@@ -60,6 +60,10 @@ categories =
     3: 10677957 # robolectric-plugin
     4: 17395619 # gradle-android-jacoco-plugin
     5: 53143469 # ios-demos
+    6: 73317406 # gradle-build-properties-plugin
+    7: 73275282 # gradle-static-analysis-plugin
+    8: 67407288 # gradle-xcodePlugin
```
Comment (blundell) [inline _coffee/application.coffee]: not valid here but shouldn't this plugin be `gradle-xcode-plugin`

File: _coffee/application.coffee [hunk 2]
```diff
@@ -134,6 +139,15 @@ categories =
 53143469 # ios-demos
 54135763 # accessibilitools
 50032733 # novoda.github.io
+2168934  # public-mvn-repo
```
Comment (blundell) [inline _coffee/application.coffee]: this is deprecated lets not add it

File: _coffee/application.coffee [hunk 3]
```diff
@@ -134,6 +139,15 @@ categories =
...
+61706237 # bonfire-firebase-sample
```
Comment (blundell) [inline _coffee/application.coffee:141]: this is a duplicate you added under apps further up

File: _coffee/application.coffee [hunk 4]
```diff
@@ -134,6 +139,15 @@ categories =
...
+73312159 # dashboards
+78538262 # eslint-config-novoda
```
Comment (blundell) [inline _coffee/application.coffee:143]: dupe

Comment (blundell) [reply _coffee/application.coffee:143]: maybe there is supposed to be dupes?

File: _coffee/application.coffee (hunk 2 above)
Comment (tasomaniac) [reply _coffee/application.coffee]: ~It is automatically added to the Others section.~ Should I add it to the `exclude` sections instead?

File: _coffee/application.coffee (hunk 4 above)
Comment (tasomaniac) [reply _coffee/application.coffee:143]: Here is like all the repos collected. I just followed the pattern. 😄 
This part is commented out, so does not have any effect on the website. 

It is not really scalable to write down every one of them.

File: _coffee/application.coffee (hunk 2 above)
Comment (tasomaniac) [reply _coffee/application.coffee]: @blundell this is already listed in `Scripting` section. Should I remove it?

File: _coffee/application.coffee (hunk 1 above)
Comment (xrigau) [reply _coffee/application.coffee]: also this seems to be a fork, I'm not sure we should include forks in these lists

Comment (tasomaniac) [reply _coffee/application.coffee]: Oh, I tried not to include forks. I thought this one was from us. Will remove it then

**Comment by tasomaniac** [comment]
@blundell @xrigau thanks for the review. 

Only question left: Should I remove `public-mvn-repo` from scripting section.

**Comment by blundell** [comment]
yes its a dead project 👍 but we just have to leave it public incase people are relying on it. But we should not advertise it for new people to use :)

File: _coffee/application.coffee [hunk 5]
```diff
@@ -96,6 +99,7 @@ categories =
     8: 41148441 # hubot-slack
     9: 52445441 # ExoPlayer
     10: 52599862 # aws-java-sample
+    11: 2168934  # public-mvn-repo
```
Comment (tasomaniac) [inline _coffee/application.coffee:102]: @blundell here, this is the excluded group.

**Review by blundell — APPROVED** [review]

**Review by xrigau — APPROVED** [review]
does this get auto-deployed on merge?

**Comment by tasomaniac** [comment]
@xrigau yes should be auto-deployed.

**Comment by tasomaniac** [comment]
auto deploy didn't work. :(
When my other PR #6 merged, it was automatically applied. I don't know what happened here. Can you guys see the difference at https://novoda.github.io/

**Pull request merged**